File: Java/Clases/src/Clase16_3_26/Clase2.java
package Clase16_3_26;
class Auto {
    String marca;
    String modelo;
    String color;
        public void EncerderAuto(){
            System.out.println("El auto " + marca + " Modelo " + modelo + " De color " + color);
        }
}
public class Clase2 {
    public static void main (String[]args){
        Auto miPrimerAuto = new Auto();
        Auto AutoEmpresa = new Auto();
        miPrimerAuto.marca= "Toyota";
        miPrimerAuto.modelo= "Corola";
        miPrimerAuto.color= "rojo";

        miPrimerAuto.EncerderAuto();
    }

}

File: Cpp/Clase16_3_26/Clase2.cpp
#include <iostream>
#include <string>

// La clase Auto representa un vehículo con marca, modelo y color.
class Auto {
public:
    // Marca del auto.
    std::string marca;
    // Modelo del auto.
    std::string modelo;
    // Color del auto.
    std::string color;

    // Método para mostrar la información del auto.
    void encenderAuto() {
        std::cout << "El auto " << marca << " Modelo " << modelo << " De color " << color << std::endl;
    }
};

// Función principal del programa.
int main() {
    // Crea dos objetos de la clase Auto.
    Auto miPrimerAuto;
    Auto autoEmpresa;

    // Asigna valores a los atributos del primer auto.
    miPrimerAuto.marca = "Toyota";
    miPrimerAuto.modelo = "Corola";
    miPrimerAuto.color = "rojo";

    // Llama al método para mostrar la información del auto.
    miPrimerAuto.encenderAuto();

    return 0;
}
